personal-portfolio/js/modal.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable id-length */
+/* eslint-disable no-magic-numbers */
+/* eslint-disable max-statements */
+/* eslint-disable func-style */
+/* eslint-disable max-len */
+/* eslint-disable sort-keys */

```

Comment:
Avoid disabling linting in your project as linters help you to write a more standard code and follow best practices.